2022-01-14 13:53:04,942 INFO    MainThread:425477 [wandb_setup.py:_flush():71] setting env: {}
2022-01-14 13:53:04,942 INFO    MainThread:425477 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-14 13:53:04,942 INFO    MainThread:425477 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-14 13:53:04,943 INFO    MainThread:425477 [wandb_init.py:_log_setup():371] Logging user logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220114_135304-2zd5w159/logs/debug.log
2022-01-14 13:53:04,943 INFO    MainThread:425477 [wandb_init.py:_log_setup():372] Logging internal logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220114_135304-2zd5w159/logs/debug-internal.log
2022-01-14 13:53:04,943 INFO    MainThread:425477 [wandb_init.py:init():404] calling init triggers
2022-01-14 13:53:04,943 INFO    MainThread:425477 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-14 13:53:04,944 INFO    MainThread:425477 [wandb_init.py:init():460] starting backend
2022-01-14 13:53:04,944 INFO    MainThread:425477 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-14 13:53:04,951 INFO    MainThread:425477 [backend.py:ensure_launched():216] starting backend process...
2022-01-14 13:53:04,954 INFO    MainThread:425477 [backend.py:ensure_launched():221] started backend process with pid: 425535
2022-01-14 13:53:04,955 INFO    MainThread:425477 [wandb_init.py:init():469] backend started and connected
2022-01-14 13:53:04,957 INFO    MainThread:425477 [wandb_init.py:init():533] updated telemetry
2022-01-14 13:53:04,966 INFO    MainThread:425477 [wandb_init.py:init():563] communicating current version
2022-01-14 13:53:05,336 INFO    MainThread:425477 [wandb_init.py:init():568] got version response 
2022-01-14 13:53:05,336 INFO    MainThread:425477 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-14 13:53:05,503 INFO    MainThread:425477 [wandb_init.py:init():606] starting run threads in backend
2022-01-14 13:53:05,556 INFO    MainThread:425477 [wandb_run.py:_console_start():1810] atexit reg
2022-01-14 13:53:05,556 INFO    MainThread:425477 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-14 13:53:05,556 INFO    MainThread:425477 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-14 13:53:05,557 INFO    MainThread:425477 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-14 13:53:05,557 INFO    MainThread:425477 [wandb_init.py:init():633] run started, returning control to user process
2022-01-14 13:53:05,775 INFO    MainThread:425477 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-14 13:53:05,776 INFO    MainThread:425477 [wandb_run.py:_restore():1752] restore
2022-01-14 13:53:08,065 INFO    MainThread:425477 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-01-14 13:53:08,249 INFO    MainThread:425477 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-01-14 13:53:08,513 INFO    MainThread:425477 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 3886
}

2022-01-14 13:53:08,614 INFO    MainThread:425477 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 3886
}

2022-01-14 13:53:08,715 INFO    MainThread:425477 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2460
  total_bytes: 3886
}

2022-01-14 13:53:08,816 INFO    MainThread:425477 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3886
  total_bytes: 3886
}

2022-01-14 13:53:08,919 INFO    MainThread:425477 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3886
  total_bytes: 3886
}

2022-01-14 13:53:09,023 INFO    MainThread:425477 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3886
  total_bytes: 3886
}

2022-01-14 13:53:09,127 INFO    MainThread:425477 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3886
  total_bytes: 3886
}

2022-01-14 13:53:09,231 INFO    MainThread:425477 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3886
  total_bytes: 3886
}

2022-01-14 13:53:09,737 INFO    MainThread:425477 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3886
  total_bytes: 3886
}

2022-01-14 13:53:10,144 INFO    MainThread:425477 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3886
  total_bytes: 3886
}
local_info {
}

2022-01-14 13:53:11,227 INFO    MainThread:425477 [wandb_run.py:_append_files():2180] logging synced files
